modules/webgl/src/classes/framebuffer.js

```diff
@@ -259,15 +259,21 @@ export default class Framebuffer extends Resource {
 
   checkStatus() {
     const {gl} = this;
-    const prevHandle = gl.bindFramebuffer(GL.FRAMEBUFFER, this.handle);
-    const status = gl.checkFramebufferStatus(GL.FRAMEBUFFER);
-    gl.bindFramebuffer(GL.FRAMEBUFFER, prevHandle || null);
+    const status = this.getStatus();
     if (status !== gl.FRAMEBUFFER_COMPLETE) {
       throw new Error(_getFrameBufferStatus(status));
     }
     return this;
   }
 
+  getStatus() {
+    const {gl} = this;
+    const prevHandle = gl.bindFramebuffer(GL.FRAMEBUFFER, this.handle);
```

Comment:
`bindFramebuffer` doesn't return anything. This logic should be removed.

Reply:
> bindFramebuffer doesn't return anything. This logic should be removed.

It is hacky, but the previous value is indeed returned, through the polyfill. It should have been mentioned in comment. 

IIRC, it was done like this to avoid fouling up global state.

Reply:
I find that very confusing. I don't think the polyfills should be introducing new behaviour.

Reply:
restored the old behavior for now, should be addressed separately https://github.com/uber/luma.gl/issues/1259